src/components/crafts-table/index.js

```diff
@@ -257,37 +264,75 @@ function CraftTable(props) {
                     return prev;
                 }, false);
 
-                if ((bestTrade && bestTrade.priceRUB > tradeData.reward.value) || (bestTrade && !includeFlea)) {
+                if ((bestTrade && bestTrade.priceRUB > tradeData.reward.sellValue) || (bestTrade && !includeFlea)) {
                     tradeData.reward.sellValue = bestTrade.priceRUB;
                     tradeData.reward.sellTo = bestTrade.vendor.name;
                 }
 
-                const priceToUse = averagePrices === true ? 'avg24hPrice' : 'lastLowPrice';
+                let fleaFeeSingle = 0;
+                let fleaFeeTotal = 0;
+                let fleaPriceToUse = craftRow.rewardItems[0].item[averagePrices === true ? 'avg24hPrice' : 'lastLowPrice'];
+                if (fleaPriceToUse === 0) {
+                    fleaPriceToUse = craftRow.rewardItems[0].item.lastLowPrice;
+                }
 
                 if (!craftRow.rewardItems[0].item.types.includes('noFlea') && (showAll || includeFlea)) {
-                    tradeData.reward.sellValue = craftRow.rewardItems[0].item[priceToUse];
+                    const bestFleaPrice = bestPrice(craftRow.rewardItems[0].item, meta?.flea?.sellOfferFeeRate, meta?.flea?.sellRequirementFeeRate);
+                    if (fleaPriceToUse === 0) {
+                        fleaPriceToUse = bestFleaPrice.bestPrice;
+                    } 
+                    fleaFeeSingle = fleaMarketFee(
+                        craftRow.rewardItems[0].item.basePrice,
+                        fleaPriceToUse,
+                        1,
+                        meta?.flea?.sellOfferFeeRate,
+                        meta?.flea?.sellRequirementFeeRate,
+                    ) * feeReduction;
+                    if (bestFleaPrice.bestPrice < fleaPriceToUse) {
+                        fleaPriceToUse = bestFleaPrice.bestPrice;
+                        fleaFeeSingle = bestFleaPrice.bestPriceFee;
+                    }
```

Comment:
Do not do the calculation if then we override it

```suggestion
                    if (bestFleaPrice.bestPrice < fleaPriceToUse) {
                        fleaPriceToUse = bestFleaPrice.bestPrice;
                        fleaFeeSingle = bestFleaPrice.bestPriceFee;
                    }
                    else {
                        fleaFeeSingle = fleaMarketFee(
                            craftRow.rewardItems[0].item.basePrice,
                            fleaPriceToUse,
                            1,
                            meta?.flea?.sellOfferFeeRate,
                            meta?.flea?.sellRequirementFeeRate,
                        ) * feeReduction;
                    }
```